src/checker/config.toml

```diff
@@ -0,0 +1,210 @@
+# Copyright Materialize, Inc. All rights reserved.
```

Comment:
To @benesch's point, could we rename this file? Maybe `chbench-config.toml` or something? That way, this code won't feel chbench-specific at all